django_datawatch/tests/test_post_delete.py

```diff
@@ -39,13 +39,12 @@ def test_update_related_calls_backend(self, mock_get_checks_for_model):
         mock_get_checks_for_model.return_value = [CheckPostDelete]
 
         # mock the manager
-        manager = mock.Mock(spec=ResultQuerySet)
-        Result.objects = manager
```

Comment:
I changed this as this was making quite the bug, it was globally affecting all tests as no scope was limited here